packages/api/lib/objectbox-model.json

```diff
@@ -0,0 +1,67 @@
+{
+  "_note1": "KEEP THIS FILE! Check it into a version control system (VCS) like git.",
+  "_note2": "ObjectBox manages crucial IDs for your object model. See docs for details.",
+  "_note3": "If you have VCS merge conflicts, you must resolve them according to ObjectBox docs.",
+  "entities": [
+    {
+      "id": "1:7075317631028539711",
+      "lastPropertyId": "8:7271446348558300706",
+      "name": "LocalTree",
+      "properties": [
+        {
+          "id": "1:596397980775022984",
+          "name": "id",
+          "type": 6,
+          "flags": 1
+        },
+        {
+          "id": "2:3646741077717461391",
+          "name": "name",
+          "type": 9
+        },
+        {
+          "id": "3:1842123839618611812",
+          "name": "address",
+          "type": 9
+        },
+        {
+          "id": "4:6237972075130416868",
+          "name": "address2",
+          "type": 9
+        },
+        {
+          "id": "5:5701425493392377207",
+          "name": "species",
+          "type": 9
+        },
+        {
+          "id": "6:318114451228785340",
+          "name": "height",
+          "type": 6
+        },
+        {
+          "id": "7:9093165262021395799",
+          "name": "circumference",
+          "type": 6
+        },
+        {
+          "id": "8:7271446348558300706",
+          "name": "readlId",
+          "type": 6
+        }
+      ],
+      "relations": []
+    }
+  ],
+  "lastEntityId": "1:7075317631028539711",
+  "lastIndexId": "0:0",
```

Comment:
Move the file into the local folder alongside the 'objectbox.g.dart'

Reply:
Done